src/core/cpuid.d

```diff
@@ -344,7 +341,7 @@ CpuFeatures cpuFeatures;
  */
 CpuFeatures* getCpuFeatures() @nogc nothrow
 {
-    pragma(inline, false);
+    version (GNU) {} else pragma(inline, false);
```

Comment:
I'd say assume you're supporting the current language version of dmd, not gdc.  Just because it doesn't support pragma inline now, doesn't mean that we should honour compatibility with 2.067 in upstream.